ui/src/components/system/Certificates.vue

```diff
@@ -113,6 +113,13 @@
                   {{$t('certificates.set_as_default')}}
                 </a>
               </li>
+              <li v-if="!props.row.default && props.row.file !== '/etc/pki/tls/certs/NSRV.crt'" role="presentation" class="divider"></li>
+              <li v-if="!props.row.default && props.row.file !== '/etc/pki/tls/certs/NSRV.crt'">
```

Comment:
Can we avoid harcoding that path one the UI?
Let's move the info on the API.

Insie the `read`, you can loop over existing certificates and add a field to JSON named like `builtin` to indicate which is the non-deletable certificate.

Reply:
the elegant way could be to use `/usr/libexec/nethserver/cert-list` from nethserver-base, what do you think ?

Reply:
We already use it and parse the output of the command inside `read`.
I guess a modification only inside the API is simpler, but you could try both ways

Reply:
Yes it is simpler, I implemented it